sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1225,6 +1249,13 @@ void _handle_search_rtr(gnrc_netif_t *netif)
     gnrc_netif_release(netif);
 }
 
+#if GNRC_IPV6_NIB_CONF_DNS
+static void _handle_rdnss_timeout(sock_udp_ep_t *dns_server)
+{
+    memset(&dns_server, 0, sizeof(sock_udp_ep_t));
```

Comment:
`dns_server` is a pointer already, hence should be `memset(dns_server, 0, sizeof(sock_udp_ep_t));`? at least on macOS I get compile error:

```
/RIOT/sys/net/gnrc/network_layer/ipv6/nib/nib.c:1255:5: error: '__builtin___memset_chk' will always overflow destination buffer [-Werror,-Wbuiltin-memcpy-chk-size]
    memset(&dns_server, 0, sizeof(sock_udp_ep_t));
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/secure/_string.h:76:3: note: expanded from macro 'memset'
                __builtin___memset_chk (dest, __VA_ARGS__, __darwin_obsz0 (dest))
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Reply:
Oops (but what did you use to compile that finds this type confusion)

Reply:
macOS -> clang

Reply:
We *really* should start having a `clang` config for the CI ;-).